README.md

```diff
@@ -1,89 +1,63 @@
- * Your own applications [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
- * [Kubeless](http://kubeless.io/) - a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
- * [SealedSecrets](https://github.com/bitnami/sealed-secrets) - Encrypt your Secret into a SealedSecret, which is safe to store - even to a public repository. 
-
-These tools are easily deployed into your cluster with just one command: ```kubeapps up``` 
-
-----
-
-## Getting started
+ * Your own application [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
+ * [Kubeless](http://kubeless.io/), a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
+ * [SealedSecrets](https://github.com/bitnami/sealed-secrets), a way to encrypt a Secret into a SealedSecret, which is safe to store...even for a public repository.
 
-To get started, read our [Getting Started guide](docs/getting-started.md).
+## Quickstart
 
-You can also browse the [full documentation](docs/).
+Kubeapps assumes a working Kubernetes (v1.7+) with RBAC enabled and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps binaries are available for both Linux and OS X, and Kubeapps has been tested with both `minikube` and Google Kubernetes Engine (GKE).
 
-----
-## Build the Kubeapps installer from source
+> On GKE, you must either be an "Owner" or have the "Container Engine Admin" role in order to install Kubeapps.
 
-The Kubeapps Installer is a CLI tool written in Go that will deploy all the Kubeapps components into your cluster.
-You can build the latest Kubeapps Installer from source. 
+The simplest way to try Kubeapps is to deploy it with the Kubeapps Installer on [minikube](https://github.com/kubernetes/minikube). 
 
-### Installing Go
-- Visit https://golang.org/dl/
-- Download the most recent Go version (here we used 1.9) and unpack the file
-- Check the installation process on https://golang.org/doc/install
-- Set the Go environment variables
+On Linux:
 
 ```
-export GOROOT=/GoDir/go
-export GOPATH=/GoDir/go
-export PATH=$GOPATH/bin:$PATH
-```
-
-### Create a working directory for the project
-
-```
-working_dir=$GOPATH/src/github.com/kubeapps/
-mkdir -p $working_dir
-```
-
-### Clone kubeapps/kubeapps repository
-
-```
-cd $working_dir
-git clone https://github.com/kubeapps/kubeapps
+minikube start
+sudo curl -s https://api.github.com/repos/kubeapps/kubeapps/releases/latest | grep linux | grep browser_download_url | cut -d '"' -f 4 | wget -i -
+sudo mv kubeapps-linux-amd64 /usr/local/bin/kubeapps
+chmod +x /usr/local/bin/kubeapps
+kubeapps up
+kubeapps dashboard
 ```
 
-### Building local binary
+On OS X:
 
 ```
-cd kubeapps
-make binary
+minikube start
```

Comment:
Same thing as above. Let's keep building the tool here, and point people to the Getting started for the rest. Also you can put it here as well if you want to, but please, let's keep the building instructions somewhere.

Reply:
Added back in README